### PR TITLE
add control plane controller

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

**Comment by anvddriesch** [comment]
@xh3b4sd the build works now. Would it make sense to already add some resources for the AZs and so on? What would be the best next steps?

File: service/controller/control_plane.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	var operatorkitController *controller.Controller
+	{
+		c := controller.Config{
+			CRD:          infrastructurev1alpha2.NewAWSControlPlaneCRD(),
+			K8sClient:    config.K8sClient,
+			Logger:       config.Logger,
+			ResourceSets: resourceSets,
+
+			// Name is used to compute finalizer names. This results in something
+			// like operatorkit.giantswarm.io/aws-operator-drainer-controller.
```
Comment (xh3b4sd) [inline service/controller/control_plane.go]: We could fix the example.

File: Gopkg.toml [hunk 2]
```diff
@@ -38,8 +38,8 @@ required = [
   branch = "master"
   name = "github.com/docker/distribution"
 
-[[constraint]]
-  branch = "master"
+[[override]]
+  branch = "hamasterspec"
   name = "github.com/giantswarm/apiextensions"
```
Comment (xh3b4sd) [inline Gopkg.toml]: Question is if we want to merge what we have in `apiextensions` and iterate from here or if we want to merge this PR and keep it on the feature branch for some time. Your call.

Comment (anvddriesch) [reply Gopkg.toml]: I'll go and request review for the `apiextensions` PR :) I don't think it's dangerous to merge it.

**Pull request merged**